scripts/openshiftci-presubmit-integration-tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# fail if some commands fails
+set -e
+# show commands
+set -x
+
+export CI="openshift"
+make configure-installer-tests-cluster
+make bin
+mkdir -p $GOPATH/bin
```

Comment:
should the `make bin` not happen after the bin directory is created?

Reply:
It does not matter.
Actually the ```$GOPATH/bin``` is created to keep the ginkgo binary within the bin dir. ```make bin``` just create the odo binary in the odo clone dir.